src/parsing/parsers/generic/jison/insert/insert.test.json

```diff
@@ -9,6 +9,16 @@
       "lowerCase": false
     }
   },
+  {
+    "namePrefix": "should not report errors",
+    "beforeCursor": "INSERT INTO bla.boo (column) VALUES (1, 2, 'a', 3); ",
```

Comment:
This is incorrect SQL :D

Let's change it to `INSERT INTO bla.boo (column) VALUES (1)`

This is the way you wanted to write it imo, but i'm not sure if it's fully supported by the parser:
`INSERT INTO bla.boo (column) VALUES (1), (2), ('a'), (3)`